workflow-templates/im-deploy-on-prem-website.yml

```
@@ -0,0 +1,460 @@
+# Workflow Code: FuzzyDragon_v1    DO NOT REMOVE
+
+# Prerequisites
+# 1 - Make sure the az secrets have been added to the environment
+# 2 - Make sure the host IIS server has been prepped to accept incoming WinRM Requests
+#     See the IIS App Pool action README for more details: https://github.com/im-open/iis-service-action/#readme
+# 3 - If you're using a deployment board, create it first: https://github.com/im-practices/git-er-done/blob/main/actions/deployment-board.md
+
+name: Deploy On Prem Website
+on:
+  workflow_dispatch:
+    inputs:
+      branch-tag-sha:
+        description: The branch, tag or sha of the code that should be deployed.
+        required: true
+      environment:
+        description: The environment to deploy to.
+        required: true
+
```

This environment variable is used in a few places in the code.  The places where the branch-tag-sha input is used can be replaced with this as well.
```suggestion
env:
  GITHUB_REF: ${{ github.event.inputs.branch-tag-sha }}
```

---

I might have gotten a little over zealous in cleaning up my env vars.